src/components/Logo/AssetLogo.tsx

```diff
@@ -76,7 +76,11 @@ export default function AssetLogo({
 }: AssetLogoProps) {
   const [src, nextSrc] = useTokenLogoSource(address, chainId, isNative, backupImg)
   const L2Icon = getChainInfo(chainId)?.circleLogoUrl
-  const [imgLoaded, setImgLoaded] = useState(false)
+  const [imgLoaded, setImgLoaded] = useState(() => {
```

Comment:
does this mean for every asset logo we'll basically be generating two images, one to be displayed and one to check if it has loaded? Does this impact page performance?

Reply:
yes. I don't think so, because although the image will exist in JS, it will not be rendered to the DOM.